qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/GatewayAuthenticationIdentityIT.java

```diff
@@ -197,13 +209,16 @@ void getTopologyRequestSucceedsWithValidAuthToken() {
   }
 
   /**
-   * Awaits the presence of the Camunda realm on the keycloak container. Once Keycloak and Identity
-   * booted up, Identity will eventually configure the Camunda Realm on Keycloak.
+   * Awaits the presence of the Camunda realm and openid keys on the keycloak container. Once
+   * Keycloak and Identity booted up, Identity will eventually configure the Camunda Realm on
+   * Keycloak.
    */
   private static void awaitCamundaRealmAvailabilityOnKeycloak() {
     final var httpClient = HttpClient.newHttpClient();
     final HttpRequest request =
-        HttpRequest.newBuilder().uri(URI.create(getKeycloakRealmAddress())).build();
+        HttpRequest.newBuilder()
+            .uri(URI.create(getKeycloakRealmAddress() + "/protocol/openid-connect/certs"))
```

Comment:
:question: Just to understand. This is the main fix. Before it was using the wrong url to check if it is ready.